Overview.bs

```diff
@@ -1036,6 +1071,27 @@ The algorithm:
 12. Return <var>entry id</var>, <var>entry</var>, <var>consumed bytes</var>, [=Mapping Entry/entryIdStringLength=] as
      <var>consumed id string bytes</var>, and <var>ignored</var>.
 
+<h5 algorithm id="remove-entries-format-2">Remove Entries from Format 2</h5>
+
+This algorithm is used to remove entries from a format 2 patch map. This removal modifies the bytes of the patch map but does not
+change the number of bytes.
+
+<dfn abstract-op>Remove Entries from Format 2 Patch Map</dfn>
+
+The inputs to this algorithm are:
+
+* <var>patch map</var>: a [=Format 2 Patch Map=] encoded patch map. May be modified by this procedure.
+
+* <var>patch URI</var>: URI for a patch which identifies the entries to be removed.
+
+This algorithm is a modified version of [$Interpret Format 2 Patch Map$], invoke [$Interpret Format 2 Patch Map$] with <var>patch map</var>
```

Comment:
The patch 1 removal description reads quite clear to me now. This still has some of the conceptual/editorial aspects that concern me. We don't have to deal with all of that in this PR but I may still want to revisit it.

Reply:
Sounds good